keps/sig-apps/2804-consolidate-workload-controllers-status/README.md

```diff
@@ -80,13 +84,14 @@ This includes defining conditions (Waiting, Running, Progressing, Available) for
 ## Motivation
 
 Today only deployment controller has a [status](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#deployment-status) to fully reflect the state during its lifecycle.
-This enhancement extends the scope of those and other conditions to other controllers (DaemonSet, Job, ReplicaSet & ReplicationController, StatefulSet).
+This enhancement extends the scope of those and other conditions to other controllers (DaemonSet, Job, ReplicaSet & ReplicationController, StatefulSet, Deployment).
```

Comment:
This motivation sounds rather incomplete.

All these objects have different policies, so it doesn't necessarily make sense that all of them have the same conditions.

What is the use case for this feature? Not saying it's not important, but I'm saying that the importance is not expressed well enough here.

Reply:
I have updated the motivation, please let me know if you are still feel it is incomplete.

These conditions do not apply for all of the controllers and this is mentioned in the proposal section: https://github.com/atiratree/kube-enhancements/blob/consolidate-status/keps/sig-apps/2804-consolidate-workload-controllers-status/README.md#proposed-conditions 

I agree it should be apparent from the beginning, so I added a `where applicable` to note it there.